### ISSUE TITLE
Dashboard.exe not starting

### ISSUE THREAD

**Issue by ODevil97** [je-can-code/rmmz-data-editor#9]
I can't get the data editor running, are there any specific install instructions I'm missing for the newer versions?
Only release Beta starts normally...

**Comment by je-can-code** [comment]
Heya pal, thanks for attempting to use this!

The application is written using [.NET8](https://dotnet.microsoft.com/en-us/download/dotnet/8.0), but its also compiled in a fashion that is supposed to be "a no-extra-installs executable"- meaning you shouldn't even need to go to that link and download it.

I'm not sure what error you're running into, so I cannot speak precisely to what your issue is.

But, taking a stab in the dark:
it may be because the application is coded presently to point at a location on disk that doesn't exist on your computer (because its pointed at _my_ computer drives- because I've been the only one to use it to date). The result being it crashes as soon as it boots because its unable to connect to the destination drive.

> The reason I believe that may be the case, is because in earlier versions, I was coding the app on the same computer I had my game project on, so it always pointed to the "C:/" drive (which almost every windows computer has- and I'm assuming you're on windows since this is a windows app). Recently, I've converted the computer I do the RMMZ development on to Linux, so now the coding for the data editor happens on my windows laptop and points to a network-mapped drive ("Z:/")- of which I have observed that if that connection is absent, the data editor won't even load up at all.

Does that sound like the issue you're encountering?

**Comment by ODevil97** [comment]
Thank you for the detailed response. 
And yes, that sounds very plausible. I could probably get around it by simply using a usb drive and assigning Z: to it then. Gonna try it if I find one lying around somewhere...

**Comment by je-can-code** [comment]
That could work as a temporary work-around to get you into the app so you can actually select where the drive should be.

It is on my `Great List of Things To Update When I Have The Time` for this app to actually create a configuration file that would instead define this initial starting location rather than using it as a hard-coded internal string in the code.

I'll revisit this post when I implement such functionality.
Thanks again for exposing my clumsy implementations 😅

**Comment by BingoEntertainment** [comment]
I have same issue. Moving the utility to drive with Z letter didn't help.

**Comment by BingoEntertainment** [comment]
I tried to install .NET8 manually, that didn't help too (I installed.NET Runtime 8.0.5, maybe I should install other version?).

**Comment by BingoEntertainment** [comment]
Anyway I still can add panels manually in json file, that's not so difficult

**Comment by je-can-code** [comment]
@BingoEntertainment 
Heya pal, I realize this issue has been outstanding for a hot minute, but I finally took some time and added a proper external configuration file that defines the project path. I did some light testing and confirmed it seems to work (in that it will pick up different project paths found in the configuration).

When you get a chance, if you're still using the plugin and this GUI/app, how about you download the latest release and give it a whirl? Here is a convenient link to the release I believe resolves this concern (but the "JMZ - zeta" release or later contains the changes required):
> https://github.com/je-can-code/rmmz-data-editor/releases/tag/autotag-9554052674-1
